apps/settings/src/components/UserList.vue

```diff
@@ -156,11 +156,12 @@
 			class="row">
 			<div id="headerAvatar" class="avatar" />
 			<div id="headerName" class="name">
-				{{ t('settings', 'Username') }}
-
 				<div class="subtitle">
-					{{ t('settings', 'Display name') }}
+					<strong>
+						{{ t('settings', 'Display name') }}
+					<strong>
```

Comment:
```suggestion
					</strong>
```
This should fix ESLint. Afterwards please run `npm run build` again and push the changes

Reply:
Oups, sorry, and fixed (+ build done)